docs/js/alert.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Alert
+layout: page.jade
+sidebar: true
+collection: js
+priority: 3
+path: alert
+---
+
+# Alert
+<p class="lead">The Alert component is an empty container where you can add a message that will be presented to users. It has an OK button by default.</p>
+
+An alert box is often used when you want to make sure information comes through to the user.
+
+### How to use
+
+You can initiate the component as a jQuery plugin:
+
+```js
+// using a data attribute
+$('[data-alert]').on('click', () => $.fn.alert(options));
+
+// using a class
+$('.data-alert').on('click', () => $.fn.alert(options));
+```
+or a vanilla constructor:
+
+```js
+import Alert from 'garden/src/js/components/alert';
+
+new Alert(options);
+```
+
+## Options
+
+Alert provides two customizable options: `textMessage` and `textButton`. By default  `textMessage` are `this is an example message` and  `textButton` are `Ok`.
```

Comment:
Acho que ficaria melhor assim: "The default valuefor the `textMessage` option is ... and for the `textButton` is ..."